conda.recipe/build.sh

```diff
@@ -2,10 +2,10 @@
 
 echo $PKG_VERSION > conda/.version
 $PYTHON setup.py install --single-version-externally-managed --record record.txt
-if [[ ! $(uname) =~ M.* ]]; then
+if [[ ! $(uname) =~ MINGW* ]]; then
   rm -rf "$SP_DIR/conda/shell/*.exe"
 fi
 $PYTHON -m conda init --install
-if [[ ! $(uname) =~ M.* ]]; then
+if [[ $(uname) =~ MINGW* ]]; then
```

Comment:
Huh, that's been broken?

Reply:
The not (`!`) was definitely breaking things. The `M*` wasn't broken, but it was less than clear what OS/distribution was being checked for, so I decided to expand it.